src/consensus/tx_verify.cpp

```diff
@@ -126,7 +126,9 @@ unsigned int GetP2SHSigOpCount(const CTransaction& tx, const CCoinsViewCache& in
     unsigned int nSigOps = 0;
```

Comment:
In commit "Restore some assert semantics in sigop cost calculations"

Maybe update commit message to suggest why policy/atmp asserts aren't added back. Not possible there, or just not worth the effort?